src/main/java/uk/gov/hmcts/probate/model/ccd/raw/request/CaseData.java

```diff
@@ -75,7 +75,7 @@
 
     // EVENT = solicitorCreateApplication
     @NotBlank(groups = {ApplicationCreatedGroup.class},
-        message = "{solsSolicitorFirmNameIsNull}")
+            message = "{solsSolicitorFirmNameIsNull}")
```

Comment:
Are you sure you have the Correct Checkstyle file in your IDE configured ? There seem to be no code changes in this file but only formatting . It should all be correct as this must have come from master . Please check again.